### PR TITLE
feat: add pagination when getting a list of data(optional)

### PR DESCRIPTION
**Pull request by ritachien**
add pagination when getting a list of data
* 前端目前沒有做分頁功能，所以除了 "前10跟隨推薦" 有預設資料筆數是 10，其他預設都是 null。這樣功能先做起來也不會影響到前端。

### CODE REVIEW

File: controllers/tweet-controller.js [hunk 1]
```diff
@@ -47,8 +47,14 @@ const tweetController = {
   },
   getTweets: async (req, res, next) => {
     try {
+      // Get query for pagination(optional)
+      const limit = Number(req.query.count) || 10
```
Comment (rubylo718) [inline controllers/tweet-controller.js]: Please remove the limit number 10 here.

**Review by rubylo718 — COMMENTED** [review]
getTweets: please remove the number

File: controllers/tweet-controller.js (hunk 1 above)
Comment (ritachien) [reply controllers/tweet-controller.js]: OK~

**Review by rubylo718 — APPROVED** [review]
Pagination is an additional feature (not included in the spec). Set the params null since this feature has not been implemented in the forestage.

**Pull request merged**